test/sys/test_pthread.rs

```diff
@@ -3,5 +3,5 @@ use nix::sys::pthread::*;
 #[test]
 fn test_pthread_self() {
     let tid = pthread_self();
-    assert!(tid > 0);
+    assert!(tid != 0);
```

Comment:
Yeah, remove it, but leave it as a separate commit with a description of why it was removed.